dev_requirements.txt

```
@@ -1 +1 @@
-zappa==0.55.0
+zappa==0.57.0
```

```suggestion
zappa~=0.60
```

I would recommend just updating to the most recent version. Though it's good you could deploy with this version!